srcpkgs/Sequeler/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'Sequeler'
+pkgname=Sequeler
```

Comment:
You could just use an uncapitalized package name to be consistent with the upstream repo and avoid repetition from having to define wrksrc manually.